src/tram/tram/management/commands/attackdata.py

```diff
@@ -20,11 +20,63 @@ def add_arguments(self, parser):
         sp_clear = sp.add_parser(CLEAR, help='Clear ATT&CK Data from the Database')  # noqa: F841
 
     def clear_attack_data(self):
-        AttackTechnique.objects.all().delete()
+        models = [AttackTechnique, AttackGroup]
+        for model in models:
+            deleted = model.objects.all().delete()
+            print(f'Deleted {deleted[0]} {model.__name__} objects')
+
+    def create_attack_technique(self, obj):
+        for external_reference in obj['external_references']:
+            if external_reference['source_name'] not in ('mitre-attack', 'mitre-pre-attack', 'mitre-mobile-attack'):
+                continue
+
+            attack_id = external_reference['external_id']
+            attack_url = external_reference['url']
+            matrix = external_reference['source_name']
+
+        assert attack_id is not None
+        assert attack_url is not None
+        assert matrix is not None
+
+        obj, created = AttackTechnique.objects.get_or_create(
+            name=obj['name'],
+            stix_id=obj['id'],
+            attack_id=attack_id,
+            attack_url=attack_url,
+            matrix=matrix
+        )
+
+        return obj, created
+
+    def create_attack_group(self, obj):
```

Comment:
Since there's only one change between the two functions `create_attack_technique` and `create_attack_group`, would it be better to coalesce them into one function with the single change?

Reply:
I had considered that, but I think that the functions will evolve independently. For instance ATT&CK Groups will probably get the `alias` portion implemented at some point. What do you think?